src/main/java/seedu/address/logic/commands/DeleteShiftCommand.java

```diff
@@ -108,22 +109,15 @@ public CommandResult execute(Model model) throws CommandException {
     }
 
     @Override
-    public boolean equals(Object other) {
-        // short circuit if same object
-        if (other == this) {
+    public boolean equals(Object o) {
+        if (this == o) {
             return true;
         }
-
-        // instanceof handles nulls
-        if (!(other instanceof DeleteShiftCommand)) {
+        if (o == null || getClass() != o.getClass()) {
```

Comment:
I think it is better to use instanceof. I do not see a reason to use getClass?
```suggestion
        if (o == null || !(o instanceof DeleteShiftCommand)) {
```